conf/analytics/analytics-code.html

```diff
@@ -0,0 +1,112 @@
+<script type="text/javascript"
+	src="/javax.faces.resource/jquery/jquery.js.xhtml?ln=primefaces&amp;v=6.2"></script>

```

Comment:
This source link to jquery.js shouldn't be necessary. As I noted in a recent comment in #5589, the placement of the custom analytics code might need to move further down the dataverse-template code, which would also resolve the need for this double reference.

Reply:
It's definitely needed now -  I get a "Uncaught ReferenceError: $ is not defined at dataset.xhtml?persistentId=doi:10.5072/FK2/JHKQQA:14" without it. Should make it easy to test...